From cfc3d8d38a36094dd5792e22c33bd758aa0e78bd Mon Sep 17 00:00:00 2001
From: Joseph Liu <kwliu@nuvoton.com>
Date: Thu, 14 Aug 2025 10:03:02 +0800
Subject: [PATCH] Set is from system interface return false

Upstream-Status: Inappropriate [oe-specific]

The ipmitool get the event is from system interface return false, but
ipmid set true as default. Change the ipmid default value to send event. Set is from system interface return false

Signed-off-by: Brian Ma <chma0@nuvoton.com>
Signed-off-by: Joseph Liu <kwliu@nuvoton.com>
---
 sensorhandler.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sensorhandler.cpp b/sensorhandler.cpp
index 7c4c5ab6..b114fd15 100644
--- a/sensorhandler.cpp
+++ b/sensorhandler.cpp
@@ -1420,7 +1420,7 @@ static bool isFromSystemChannel()
     // TODO we could not figure out where the request is from based on IPMI
     // command handler parameters. because of it, we can not differentiate
     // request from SMS/SMM or IPMB channel
-    return true;
+    return false;
 }
 
 ipmi::RspType<> ipmicmdPlatformEvent(ipmi::Context::ptr& ctx,
-- 
2.34.1

